Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -22,6 +22,9 @@ class Setting: NSObject {
 
     weak var delegate: SettingsDelegate?
 
+    // For wrapping footer titles, double the footer height. A better solution would be to calculate the height needed.
+    var isFooterDoubleHeight = false
```

Comment:
Currently I am trying to replace this with  attributed text's boundingRect(with: frameSize…